### PR TITLE
ADS: Tab Switcher

### PR DESCRIPTION
**Pull request by malmstein**
Task/Issue URL:  https://app.asana.com/0/1174433894299346/1203411647299664
Design Review Task: https://app.asana.com/0/0/1203569245422155/f

### Description
Update Tab switcher screen

### CODE REVIEW

File: app/src/main/res/layout/item_tab.xml [hunk 1]
```diff
@@ -67,24 +64,20 @@
             android:background="?selectableItemBackground"
             android:contentDescription="@string/closeContentDescription"
             android:scaleType="center"
-            android:src="@drawable/ic_close_tab"
-            android:tint="?normalTextColor"
+            android:src="@drawable/ic_close_24"
```
Comment (nalcalag) [inline app/src/main/res/layout/item_tab.xml:67]: I think this icon should be `Close-24-small` that need to be imported in the project

**Review by nalcalag — APPROVED** [review]
Approved! I think the close icon in the tab items should be `ic_close_24_small` but I suppose it will show up in the design review if I'm not wrong 👍

**Comment by aitorvs** [comment]
Current dependencies on/for this PR:
* feature/david/ads_release
  * **PR #2657** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2657" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/2657?utm_source=stack-comment).

File: app/src/main/res/layout/item_tab.xml (hunk 1 above)
Comment (malmstein) [reply app/src/main/res/layout/item_tab.xml:67]: All good!

**Pull request merged**